test/cfg/runtests.sh

```diff
@@ -449,26 +449,7 @@ function cppunit_fn {
 
 # selinux.c
 function selinux_fn {
-    if [ $HAS_PKG_CONFIG -eq 1 ]; then
```

Comment:
Why did you remove this?

Reply:
because libselinux does not make use of pkg-config